R/utility.R

```diff
@@ -170,13 +170,11 @@ createOutputFile <- function(filePattern, url) {
   )
 
   # Parsing url to obtain container's virtual directory path
-  azureDomain <- "blob.core.windows.net"
-  parsedValue <- strsplit(url, azureDomain)[[1]]
+  parsedValue <- strsplit(url, "/")[[1]]
 
-  accountName <- parsedValue[1]
-  urlPath <- parsedValue[2]
+  urlPath <- paste0("/", parsedValue[4])
 
-  baseUrl <- paste0(accountName, azureDomain)
+  baseUrl <- paste0(parsedValue[1], "//", parsedValue[3])
```

Comment:
nitpick: Since you removed the named varailbes for the array indexes, can you add a comment or something to say what parsedValue[1] and parsedValue[3] are?